src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -5455,6 +5459,164 @@ namespace ZoneEquipmentManager {
 		MassConservation(ZoneNum).MixingSourceMassFlowRate = ZoneSourceMassFlowRate;
 	}
 
+	void
```

Comment:
This is a nice, packaged up, routine with a clear purpose.  Easy to unit test.  This is good.  I have only three comments, all minor, and more just comments for future coding than this branch.
 - For some reason, the function name indentation is 2 tabs deep, not sure why.  If you happen to go back into the code for any changes, just take a tab out and it will fall in line with the rest of the code.
 - I would take out the unused portions of the function comment header.  It doesn't hurt that they are there, and since you have included a note in the REFERENCES section, I'd leave that.  Just the unused na ones dont need to be included.
 - This would have been ripe for a very tiny move toward object oriented coding.  Here's how it would have been done:

`ZoneSizingInput` is an array of type `ZoneSizingInputData`.  If you go over to that struct definition, you could add a function declaration like this:

```
struct ZoneSizingInputData {
  // lots of stuff
  // variables, etc.
  void sizeDOASControlStrategy();
  // other stuff
}
```

Now there would be a function called sizeDOASControlStrategy that is a _member_ of the struct.  To call this function, you would then loop over the instances and call that method:

```
for ( auto & thisZoneData : ZoneSizingInput ) {
  thisZoneData.sizeDOASControlStrategy();
}
```

So far so good.  Then inside `sizeDOASControlStrategy`, you would just operate on the current instance:

```
void ZoneSizingInputData::sizeDOASControlStrategy() {
  // comments
  if ( this->AccountForDOAS ) {
    if ( this->DOASControlStrategy == DOASNeutralSup ) {
      if ( this->DOASLowSetpoint == AutoSize && this->DOASHighSetpoint == AutoSize ) {
        this->DOASLowSetpoint = 21.1;
        this->DOASHighSetpoint = 23.9;
      }
  // etc...
```

Inside that function, you are always operating on the "current zone sizing input", or "THIS zone sizing input".  That's the purpose of the `this->`, which is actually optional.  

This would be a small change, but would be one step closer to objectification, and even easier to unit test, in my opinion.  And it's also a form of "gateway" into OO since it is such an isolated change.

OK, that is all, thanks.